<?xml version="1.0"?>

<launch>

    <param name="/use_sim_time" type="bool" value="true"/>

    <!-- Stage -->
    <node pkg="stage_ros" type="stageros" name="stageros" required="true"
          args="-d $(find particle_filter_localisation)/config/map.world"/>

    <!-- Map server -->
    <node pkg="map_server" type="map_server" name="map_server" required="true"
          args="$(find particle_filter_localisation)/config/map.yaml"/>

    <!-- RViz 
-->
    <node pkg="rviz" type="rviz" name="rviz" required="true"
          args="-d $(find particle_filter_localisation)/config/visualisation.rviz"/>
    <!-- Particle filter localisation -->
    <node pkg="particle_filter_localisation" type="particle_filter_localisation" name="particle_filter_localisation"
          output="screen">

         <param name="num_particles" type="int" value="1000"/> 
         <param name="num_motion_updates" type="int" value="1"/> 
         <param name="num_scan_rays" type="int" value="6"/> -->
         <param name="num_sensing_updates" type="int" value="1"/>

         <param name="motion_distance_noise_stddev" type="double" value="0.01"/> 
         <param name="motion_rotation_noise_stddev" type="double" value="0.052360"/> 
         <param name="sensing_noise_stddev" type="double" value="0.5"/> 

    </node>

</launch>
